src/search.cpp

```diff
@@ -81,6 +81,7 @@ setup_queryParser(Xapian::QueryParser* queryparser,
                   Xapian::Database& database,
                   const std::string& language,
                   const std::string& stopwords,
```

Comment:
To me this is not clear should be camel case or not? But having two different ways to represent variables is not recommended.